tests/test_metrics.py

```diff
@@ -46,7 +45,19 @@ def test_failures(project):
     assert (
         exception_caught == True
     ), "LenaMetrics failed to throw an exception despite an invalid ITS set being provided"
+    
     exception_caught = False
+    try:
+        lm = pd.DataFrame(np.array(
+            [["voc_speaker","segments_vtc",'FEM'],         
+             ]), columns=["callable","set","speaker"])
+        m = Metrics(project, lm,  segments="unknown")
+    except ValueError as e:
+        exception_caught = True
+
+    assert (
+        exception_caught == True
+    ), "Metrics failed to throw an exception despite having the segments argument and by having a value different than 'recording_filename'"
```

Comment:
oh this is where checks happen, is it??

Reply:
and it even checks for recs being valid -- cool!

Reply:
or maybe not? maybe this is just for the test with the data provided below, right?

Reply:
This is indeed just the automated tests to make sure the command fails when the arguments are not correct.
But the check for recordings being in the dataset is actually done during the extraction here:
https://github.com/LAAC-LSCP/ChildProject/blob/e3dc054e968d4217b60a9b384c205907d2bfc3f7/ChildProject/pipelines/metrics.py#L317